src/renderers/webgl/WebGLTextures.js

```diff
@@ -671,7 +671,7 @@ function WebGLTextures( _gl, extensions, state, properties, capabilities, utils,
 
```

Comment:
Further down in this file, do we need to add `wrapR` here?

```javascript
	function getTextureCacheKey( texture ) {

		const array = [];

		array.push( texture.wrapS );
		array.push( texture.wrapT );
		array.push( texture.magFilter );
		array.push( texture.minFilter );
		// ...
```

Reply:
Good catch! I guess `array.push( texture.wrapR || 0 );` should do it.

Reply:
Updated!